### ISSUE TITLE
Connections into spa.BasalGanglia should be owned by spa.SPA, not the BG

### ISSUE THREAD

**Issue by tcstewar** [nengo/nengo#776]
In this model, the Connection from vision to BG is stored in model.bg.  This is fine for the Simulation, but is a bit weird for the visual display, since it means the lines aren't drawn unless the BG network is expanded

``` python
import nengo
import nengo.spa as spa

D = 32  # the dimensionality of the vectors

model = spa.SPA()
with model:
    model.vision = spa.Buffer(D)

    model.speech = spa.Buffer(D)    

    actions = spa.Actions(
        'dot(vision, DOG) --> speech=BARK',
        'dot(vision, CAT) --> speech=MEOW',
        'dot(vision, RAT) --> speech=SQUEAK',
        'dot(vision, COW) --> speech=MOO',
        )

    model.bg = spa.BasalGanglia(actions)
    model.thal = spa.Thalamus(model.bg)
```

**Comment by Seanny123** [comment]
How would you go about changing this?

**Comment by tcstewar** [comment]
Hmm...  now that I look into the code, I'm wondering whether this is actually some other bug.  I'd expected it to just be a missing ```with self.spa:``` but I don't see one missing.  :(

**Comment by jgosmann** [comment]
This seems to be another thing fixed with #1137. :)
